app/views/statistics/index.html.erb

```diff
@@ -1,6 +1,12 @@
 <% title _("Statistics"), _('Statistics') %>
-<div id="statistics">
-<% charts.each do |ch| %>
-  <%= content_tag(:div, ch, :class=>"stats-well") %>
-<% end %>
-</div>
+
+<div id="statistics"></div>
+
+<%= javascript_tag defer: 'defer' do -%>
```

Comment:
I originally implemented this as a separate file, managed by the asset pipeline but not as part of the main application.js file. I wouldn't include this code in the bundled webpack file as it contains a handler to a global event. This file should only be loaded when needed as the handler would otherwise be called on every page.

Generally using inline script like I did here is considered a bad practice. I decide to implement this way for simplicity's sake since the file would contain only one line. The advantage is that we don't have to create another file or add the code to include it in the asset pipeline. I think we  will appreciate the simplicity as we add more client side functionality to other pages in the future.